apps/condo/domains/organization/integrations/sbbol/routes.js

```diff
@@ -39,7 +39,8 @@ class SbbolRoutes {
         const reqId = req.id
         const sbbolAuthApi = await initializeSbbolAuthApi()
         try {
-            if (!isObject(req.session[SBBOL_SESSION_KEY].checks)) {
+            const checks = get(req, ['session', SBBOL_SESSION_KEY, 'checks'])
+            if (!isObject(checks)) {
                 logger.info({ msg: 'SBBOL invalid nonce and state', reqId })
                 return res.status(400).send('ERROR: Invalid nonce and state')
```

Comment:
Invalid or empty?

Reply:
Maybe, but I don't think it's that important. We generate them and put them in the session, that is, we are sure that they are definitely there. Deleting a session is not standard behavior for a user